app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :post
+
+  after_save :update_likes_counter
+
+  def update_likes_counter
+    post.increment!(:likes_counter)
+  end
```

Comment:
[OPTIONAL]
- Same here. I would recommend making the `update_likes_counter` private method since we are calling this method only from inside the class and with this it is more secure.